### PR TITLE
Safari margin - Tab

### PR DESCRIPTION
**Pull request by laraharrow**
### :sparkles: Changes

- Update Tab to remove preset margin on Safari

### :white_check_mark: Requirements

- [ ] Includes test coverage for all changes
- [ ] Build and tests are passing
- [ ] Update documentation
- [ ] Updated CHANGELOG
- [ ] Checked for i18n impacts
- [ ] Checked for a11y impacts
- [ ] Check for image-snapshot changes (run `yarn image-snapshots` locally)
- [ ] PR is ideally < ~400LOC

### :camera: Screenshots

Screen Shot 2020-12-07 at 4 59 17 PM

### CODE REVIEW

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Tab's extra margin on Safari
```
Comment (lukelooker) [inline packages/components/CHANGELOG.md:28]:
```suggestion
- `Tab` now displays properly in Safari
```

File: packages/components/src/Tabs/Tab.tsx [hunk 2]
```diff
@@ -68,7 +68,8 @@ const TabStyle = styled.button<TabProps>`
     selected ? theme.colors.text5 : theme.colors.text2};
   cursor: pointer;
   font-family: ${({ theme }) => theme.fonts.brand};
-
+  /* this is necessary to avoid safari automatic margin */
+  margin: 0;
   &:active {
```
Comment (lukelooker) [inline packages/components/src/Tabs/Tab.tsx:73]:
```suggestion
  /* Remove default margin button in Safari */
  margin: 0;
  
  &:active {
```

**Review by lukelooker — COMMENTED** [review]
The snapshot file at `packages/components/snapshots/Tabs/Controlled-snap.png` should get removed since the matching story was removed.

Two very minor documentation tweaks noted as well.

**Comment by laraharrow** [comment]
hey @lukelooker I fixed the documentation that you suggested. But as far as `packages/components/snapshots/Tabs/Controlled-snap.png` -  that story wasn't removed. All I did was change from the file below under the `storybook` folder to the `Tab` folder and now its under `Tabs.story.tsx`

File: packages/components/src/Tabs/Tabs.story.tsx [hunk 3]
```diff
@@ -61,13 +55,36 @@ const Template: Story<DemoProps> = ({ tabCount, tabPrefix, ...args }) => {
...
 }
 
-export const Primary = Template.bind({})
-Primary.args = {
+export const Basic = Template.bind({})
+Basic.args = {
   distribute: true,
   tabCount: 20,
   tabPrefix: 'My Awesome Tab',
 }
```
Comment (laraharrow) [inline packages/components/src/Tabs/Tabs.story.tsx:64]: @lukelooker this is where the controlled story lives now

**Review by lukelooker — DISMISSED** [review]
Ah... sorry about that.

LGTM! :)

**Review by lukelooker — APPROVED** [review]
LGTM

**Pull request merged**